content/news/2024-02-03-bevy-0.13/index.md

```diff
@@ -1284,7 +1284,16 @@ Check out the [**Bevy 0.14 Milestone**](https://github.com/bevyengine/bevy/miles
 * **More editor experimentation:** TODO
 * **bevy_dev_tools:** TODO
 * **A revised scene format:** TODO
-* **bevy_ui improvements:** TODO
+* **bevy_ui improvements:** `bevy_ui` has its fair share of problems and limitations, [both mundane and architectural].
```

Comment:
the lack of capitalization in the bullet point irks me, maybe reorder so you can do 'Improving bevy_ui', but this is a nit.

Reply:
I think we should we subheadings rather than bullet points for this section, but I'm going to do that in the main PR to avoid merge conflicts.